deploy/services-demo/conf/nginz/nginx.conf

```diff
@@ -311,6 +311,11 @@ http {
       proxy_pass http://galley;
     }
 
+    location ~* ^/conversations/([^/]*)/([^/]*)/otr/messages {
```

Comment:
Heh; this `services_demo` nginx config is much more readable than the  yaml mess. makes you think :)